crates/client/assets/schema.sdl

```diff
@@ -1338,7 +1338,11 @@ type Subscription {
 		"""
 		The ID of the transaction
 		"""
-		id: TransactionId!
+		id: TransactionId!,
+		"""
+		If true, accept to receive the preconfirmation status
+		"""
+		allowPreconfirmation: Boolean
```

Comment:
nit: I'd prefer either `includePrecofirmations` or just `preconfirmations`

Reply:
Changed to `includePrecomfirmation` it's still singular as you can only receive 1 preconfirmation for a TX.